go/vt/vttablet/onlineddl/vrepl/columns_test.go

```diff
@@ -133,3 +133,248 @@ func TestGetSharedColumns(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
This added test is mostly irrelevant to the changes in this PR, but while writing the PR I noticed how this was missing.